upgrade/upgrade-1.1.0.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * 2020-present Friends of Presta community
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the MIT License
+ * that is bundled with this package in the file LICENSE
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/MIT
+ *
+ * @author    Friends of Presta community
+ * @copyright 2020-present Friends of Presta community
+ * @license   https://opensource.org/licenses/MIT MIT
+ */
+/**
+ * /views/css/front.css is under cvs, it should not be.
+ * It's shop specific.
+ * It must not be overwritten by module upgrade.
+ *
+ * Copy current file contents to new file
+ */
+function upgrade_module_1_1_0($module)
+{
+    // @todo use dirname(__FILE__) instead of __DIR__ ?
```

Comment:
The process is ok for me can we remove the @todo ?

Reply:
yes, I'll add a .gitignore file with views/css/fop_customcss.css